cmd/submit.go

```diff
@@ -164,16 +222,19 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 
         %s download --exercise=%s --track=%s
 
-		`
-		return fmt.Errorf(msg, BinaryName, metadata.Exercise, metadata.Track)
+        `
+		return nil, fmt.Errorf(msg, BinaryName, metadata.Exercise, metadata.Track)
 	}
+	return metadata, nil
+}
 
-	exercise.Documents = make([]workspace.Document, 0, len(args))
-	for _, file := range args {
+func (s *submitCmdContext) _documents(filepaths []string, exercise workspace.Exercise) ([]workspace.Document, error) {
```

Comment:
It's just a workaround to avoid errors due to naming clashes (there is a `documents()` method that is saved as a `documents` field). IIRC, there was previous mention of avoiding "getter" type naming for methods so using an underscore seemed like a reasonable alternative.

Reply:
I suspect that if there's a naming clash, there's likely a problem with the abstraction, not really with the naming convention.